docs/install.md

```diff
@@ -132,28 +153,65 @@ blanks the `myradio_test` database each time it is ran, so it can be used to
 reset the database and config file, should this prove necessary.
 
 Summary:
-* `composer install`
-* `vagrant up`
-* `vagrant ssh -- /vagrant/scripts/reset-db.sh`
-* `src/vendor/bin/codecept run`
+
+-   `composer install`
+-   `vagrant up`
+-   `vagrant ssh -- /vagrant/scripts/reset-db.sh`
+-   `src/vendor/bin/codecept run`
 
 The vagrant initialisation script also runs `composer install`, but that is run
 on the virtual machine which also installs the PHP extensions required for
 Codeception. These extensions may be missing locally, so running composer will
 confirm that they are present.
 
 ## Next Steps
+
 Once you've got through the setup wizard, the next thing that's most useful to
 you is most likely creating a show.
 
 To do this, you first need to:
-- Create a Term (Show Scheduler -> Manage Terms)
-- Create a Show (List My Shows -> Create a Show)
-- Apply for a Season of your new Show (List My Shows -> New Season)
-- Schedule the Season (Shows Scheduler)
+
+-   Create a Term (Show Scheduler -> Manage Terms)
+-   Create a Show (List My Shows -> Create a Show)
+-   Apply for a Season of your new Show (List My Shows -> New Season)
+-   Schedule the Season (Shows Scheduler)
+
+### Setting up your own 2016-site
```

Comment:
Shouldn't this be in the 2016-site repo?

Reply:
I was going based on james suggestion of adding troubleshooting to myradio docs. But mainly I figured if your trying to connect 2016-site and myradio, you're likely troubleshooting myradio rather 2016-site, however we could link this on 2016-site repo

Reply:
it shouldn't be here. otherwise it becomes, why doesn't myradio have instructions for how to set up all these other 10 things, cos they all use myradio

Reply:
that is a fair point, should I take a copy of this and pr it over 2016-site